app/models/user.rb

```diff
@@ -4,7 +4,7 @@ class User < ApplicationRecord
   has_many :likes, foreign_key: 'author_id'
 
   validates :name, presence: true
-  validates :posts_counter, numericality: { only_integer: true }, comparison: { greater_than_or_equal_to: 0 }
+  # validates :posts_counter, numericality: { only_integer: true }, comparison: { greater_than_or_equal_to: 0 }
```

Comment:
- Kindly remove all commented code for a more cleaner and readable code 👍🏽

Reply:
@rotimiazeez 
worked on them and thank you